helm/ignition-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,35 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "ignition-operator.name" -}}
+{{- .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "ignition-operator.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "ignition-operator.labels" -}}
+app: {{ include "ignition-operator.name" . | quote }}
+{{ include "ignition-operator.selectorLabels" . }}
```

Comment:
Let's put this line on the top or bottom of `define` block

Reply:
So I think the question is do we want to have a nice order in the source, or in the rendered output (i.e. in labels on the actual objects in k8s), because this is currently mostly sorted alphabetically in source (except that `app` which was removed and then restored and just ended up in wrong place) which renders as:

```
app: "ignition-operator"
app.kubernetes.io/name: "ignition-operator"
app.kubernetes.io/instance: "ignition-operator-1.0.0"
app.giantswarm.io/branch: "voo-ensure-labels"
app.giantswarm.io/commit: "9be65811cbc0b779d36522970f4291f6fd0850df"
app.kubernetes.io/managed-by: "Helm"
app.kubernetes.io/version: "0.0.1"
helm.sh/chart: "ignition-operator-0.0.1-9be65811cbc0b779d36522970f4291f6fd0850d"
```

which isn't great. Putting the include at the top would result in:

```
app.kubernetes.io/name: "ignition-operator"
app.kubernetes.io/instance: "ignition-operator-1.0.0"
app: "ignition-operator"
app.giantswarm.io/branch: "voo-ensure-labels"
app.giantswarm.io/commit: "9be65811cbc0b779d36522970f4291f6fd0850df"
app.kubernetes.io/managed-by: "Helm"
app.kubernetes.io/version: "0.0.1"
helm.sh/chart: "ignition-operator-0.0.1-9be65811cbc0b779d36522970f4291f6fd0850d"
```

which is worse. Putting at the bottom:

```
app: "ignition-operator"
app.giantswarm.io/branch: "voo-ensure-labels"
app.giantswarm.io/commit: "9be65811cbc0b779d36522970f4291f6fd0850df"
app.kubernetes.io/managed-by: "Helm"
app.kubernetes.io/version: "0.0.1"
helm.sh/chart: "ignition-operator-0.0.1-9be65811cbc0b779d36522970f4291f6fd0850d"
app.kubernetes.io/name: "ignition-operator"
app.kubernetes.io/instance: "ignition-operator-1.0.0"
```

probably just as bad? So I'd actually put it in the middle:

```
app: {{ include "ignition-operator.name" . | quote }}
app.giantswarm.io/branch: {{ .Values.project.branch | quote }}
app.giantswarm.io/commit: {{ .Values.project.commit | quote }}
{{ include "ignition-operator.selectorLabels" . }}
app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
helm.sh/chart: {{ include "ignition-operator.chart" . | quote }}
```

to have this result:

```
app: "ignition-operator"
app.giantswarm.io/branch: "voo-ensure-labels"
app.giantswarm.io/commit: "9be65811cbc0b779d36522970f4291f6fd0850df"
app.kubernetes.io/name: "ignition-operator"
app.kubernetes.io/instance: "ignition-operator-1.0.0"
app.kubernetes.io/managed-by: "Helm"
app.kubernetes.io/version: "0.0.1"
helm.sh/chart: "ignition-operator-0.0.1-9be65811cbc0b779d36522970f4291f6fd0850d"
```

WDYT?

Reply:
TBH I don't care :D I want it to be sorted here so it's straight forward to add a new line when _coding_. Making sure everything is ordered after rendering is unreliable/impossible anyway.

Some pseudocode to prove my point:

```
{{ define name }}
a
z
{{ end }}

b
c
{{ include name }}
```

Reply:
Yeah OK, that's true 👍